sdk/cosmos/azure-cosmos/azure/cosmos/_synchronized_request.py

```diff
@@ -90,10 +90,7 @@ def _Request(global_endpoint_manager, request_params, connection_policy, pipelin
     """
     # pylint: disable=protected-access
 
-    is_media = request.url.find("media") > -1
-    is_media_stream = is_media and connection_policy.MediaReadMode == documents.MediaReadMode.Streamed
```

Comment:
We should remove the MediaReadMode enum from documents.py

Reply:
we should also move all media related api s on cosmos_client_connection, will do that in the next iteration